ark/mibi/qc_comp.py

```diff
@@ -1,14 +1,169 @@
 import copy
+import matplotlib.pyplot as plt
+import os
 import numpy as np
 import pandas as pd
-import matplotlib.pyplot as plt
+from requests.exceptions import HTTPError
 from scipy.ndimage import gaussian_filter
 import seaborn as sns
+from shutil import rmtree
+from skimage.io import imsave
 
+from ark.mibi.mibitracker_utils import MibiRequests
+
+import ark.settings as settings
 import ark.utils.io_utils as io_utils
 import ark.utils.load_utils as load_utils
 import ark.utils.misc_utils as misc_utils
 
+# needed to prevent UserWarning: low contrast image barf when saving images
+import warnings
+warnings.filterwarnings('ignore')
+
+
+def create_mibitracker_request_helper(email, password):
+    """Create a mibitracker request helper to access a user's MIBItracker info on Ionpath
+
+    Args:
+        email (str):
+            The user's MIBItracker email address
+        password (str):
+            The user's MIBItracker password
+
+    Returns:
+        ark.mibi.mibitracker_utils.MibiRequests:
+            A request helper module instance to access a user's MIBItracker info
+    """
+
+    try:
+        return MibiRequests(settings.MIBITRACKER_BACKEND, email, password)
+    except HTTPError:
+        print("Invalid MIBItracker email or password provided")
```

Comment:
does the `HTTPError` here contain info on whether it was the email or pw that was wrong?

Reply:
It doesn't unfortunately.